packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/instance/arrowAngle.st

```diff
@@ -0,0 +1,4 @@
+geometry
+arrowAngle
+
+	^ 135
```

Comment:
There is an empty line there. If there were none, there would be a sign indicating no new line.